ngraph_bridge/ngraph_utils.h

```diff
@@ -27,13 +27,16 @@
 #include "tensorflow/core/platform/tensor_coding.h"
 #include "tensorflow/core/util/saved_tensor_slice_util.h"
 
-#include "ngraph/event_tracing.hpp"
 #include "ngraph/ngraph.hpp"
+#include "ngraph/runtime/chrome_trace.hpp"
 #include "ngraph/serializer.hpp"
 
 #include "logging/ngraph_log.h"
 #include "logging/tf_graph_writer.h"
 
+#define NG_TRACE(name, category, args) \
+  ngraph::runtime::event::Duration dx__ { name, category, args }
```

Comment:
Also from good macro writing practice point of view, maybe we wrap these in brackets: `ngraph::runtime::event::Duration dx__ { (name), (category), (args) }`